roles/openshift_facts/tasks/main.yml

```diff
@@ -10,12 +10,19 @@
 - set_fact:
     l_is_containerized: "{{ (l_is_atomic | bool) or (containerized | default(false) | bool) }}"
 
-- name: Ensure PyYaml is installed
-  action: "{{ ansible_pkg_mgr }} name=PyYAML state=present"
+- name: Ensure required packages are installed
+  action: "{{ ansible_pkg_mgr }} name={{ item }} state=present"
+  with_items:
+    - PyYAML
+    - yum-utils
+    - dbus-python
```

Comment:
That will install python 2 version of the dependencies, so I would suggest doing something like this:
```
    with_items:
    - "libsemanage-python{{ ansible_python['version']['major']}}"
    - "python{{ ansible_python['version']['major']}}-dbus"
```

So this can be safely run on python 3 (I am currently testing ansible on python 3, and I was able to deploy on F24 with python 3 installed and used with ansible 2.2)